application/src/main/java/org/togetherjava/tjbot/features/basic/ChatGPTFormatter.java

```diff
@@ -0,0 +1,31 @@
+package org.togetherjava.tjbot.features.basic;
+
+import org.togetherjava.tjbot.features.chatgpt.ChatGptService;
+
+import java.util.Optional;
+
+/**
+ * A utility class for formatting code using ChatGPT.
```

Comment:
this class should be in `code` package instead since `basic` is for example commands only imo